chain/chain/src/test_utils.rs

```diff
@@ -831,6 +831,10 @@ impl RuntimeAdapter for KeyValueRuntime {
         }
     }
 
+    fn get_gc_head_height(&self, _block_hash: &CryptoHash) -> Result<BlockHeight, Error> {
+        Ok(0)
```

Comment:
`get_gc_head_height` should be turned on for KV runtime
To do that, use the formula `block height - NUM_EPOCHS_TO_KEEP_STORE_DATA * epoch length`